shopify_app.gemspec

```diff
@@ -8,9 +8,7 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6', '< 5.0')
```

Comment:
well only the Gemfiles that override it would have it. My thought is: `gemspec` should always be the main source of dependencies (when possible). The only reason we have other separate Gemfiles is for testing purposes.